ako-operator/api/v1alpha1/akoconfig_types.go

```diff
@@ -183,7 +183,9 @@ type AKOConfigSpec struct {
 	// ImageRepository is where the AKO controller resides.
 	ImageRepository string `json:"imageRepository,omitempty"`
 	// ImagePullPolicy defines when the AKO controller image gets pulled.
-	ImagePullPolicy       string `json:"imagePullPolicy,omitempty"`
+	ImagePullPolicy string `json:"imagePullPolicy,omitempty"`
+	// ReplicaCount defines the number of replicas for AKO Statefulset
+	ReplicaCount          int `json:"replicaCount,omitempty"`
```

Comment:
@akshayhavile @DixitAakash Do we need to expose the `securityContext` and `podSecurityContext` also?

Reply:
@swathinsankaran based on discussions we won't be exposing **securityContext** and **podSecurityContext** currently. @akshayhavile can you please confirm?